evap/results/templates/result_bar.html

```diff
@@ -21,7 +21,11 @@
 >
 
     <div class="grade-bg-result-bar-count text-center{% if question_result.counts is None %} grade-bg-disabled{% endif %}">
-        <span class="fas fa-user small"></span> {{ question_result.total_count|default:0 }}
+        {% if question_result.total_count %}
+            <span class="fas fa-user small"></span> {{ question_result.total_count }}
+        {% else %}
+            strich goes here
```

Comment:
please use `&mdash;` for the dash

Reply:
actually this will now also show a dash for questions where the number of voters is 0 even if other questions received enough answers to publish the results. i think in this case we should show the 0, don't you think? so use the dash only if `can_publish_rating_results` is `False`.

example:
![novoter](https://user-images.githubusercontent.com/1781719/41934325-bf08ab5c-7986-11e8-8499-2a091fdceed7.PNG)